docs/src/test-runners-python.md

```diff
@@ -63,6 +63,19 @@ def test_my_app_is_working(fixture_name):
 - `browser_type_launch_args`: Override launch arguments for [`method: BrowserType.launch`]. It should return a Dict.
 - `browser_context_args`: Override the options for [`method: Browser.newContext`]. It should return a Dict.
 
+## Parallelism: Running Multiple Tests at Once
```

Comment:
In the future, we can improve even more by running multiple tests/contexts at once within a worker like we do in official Test Runner, but this at least improves runtime for machines that have multiple CPUs and are currently just doing everything serially.